unit_test/fuzzing/test_transport/test_spdm_transport_pci_doe_encode_message/spdm_transport_pci_doe_encode_message.c

```diff
@@ -39,7 +43,10 @@ void libspdm_test_transport_pci_doe_encode_message(void **State)
                              0; /* PCI_DOE_MAX_RANDOM_NUMBER_COUNT */
     LIBSPDM_ASSERT(spdm_test_context->test_buffer_size > record_header_max_size);
 
-    transport_message_size = spdm_test_context->test_buffer_size - record_header_max_size;
+    alignment = PCI_DOE_ALIGNMENT;
+    aligned_message_size = (spdm_test_context->test_buffer_size + (alignment - 1)) &
+                           ~(alignment - 1);
+    transport_message_size = aligned_message_size;
```

Comment:
please explain why

Reply:
There are alignment operations in the function
https://github.com/DMTF/libspdm/blob/345e2f866ca463f989343a126e7beb6a563d7c31/library/spdm_transport_mctp_lib/libspdm_mctp_mctp.c#L71-L76